tools/clang/unittests/HLSL/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ add_clang_library(ClangHLSLTests SHARED
   PixTestUtils.cpp
   RewriterTest.cpp
   SystemValueTest.cpp
+  ${CMAKE_SOURCE_DIR}/lib/DxcSupport/Unicode.cpp
```

Comment:
Are you sure this is needed?  I would have expected that we get this through the dxcsupport lib.  I think this should have been included by the `set( LLVM_COMPONENTS ...)` section earlier.